opm/simulators/wells/MSWellHelpers.hpp

```diff
@@ -206,7 +206,7 @@ namespace mswellhelpers
     template <typename ValueType>
     ValueType velocityHead(const double area, const ValueType& mass_rate, const ValueType& density)
     {
-        return (0.5 * mass_rate * mass_rate / (area * area * density));
```

Comment:
What is the reason to remove `0.5` here?

Reply:
If you found out that the formulations was wrong. It will be worthwhile to add a comment here for later reference.

Reply:
Strange, I thought I added a comment here. I think the reasoning is similar as for  the "2" in the friction term.

Reply:
Then please test it out with the commercial simulator. We keep the `0.5` for some formulations, like `valveContrictionPressureLoss`.  

Friction and acceleration are different properties of forces coming with two different formulations, not sure we can assume both formulations are wrong at the same time in case it was an accident typo in one of the formulations.

Reply:
> Then please test it out with the commercial simulator. 
That's what I did.

Reply:
> That's what I did.

That is very good. I do not have any reference example can check this value, it is very good you can look into it. 

But it comes a little bit surprise, because there is 0.5 \rho v^2 in Bernoulli's equation, not sure why the 0.5 is omitted here.

Reply:
One of the possible explanations is that the unit conversion factor (similar to frictional pressure loss, which also has the wrong conversion factor) is wrong, it forces us to match the mistake. 

Since we are using SI unit, it is much easier to program with the correct formulation, since there is no unit conversion involved.